docs/cluster_spec.md

```diff
@@ -249,6 +249,43 @@ spec:
       image: busybox
 ```
 
+### fileAssets
+
+FileAssets permit you to place inline file content into the cluster and instanceGroup specification, which can be consumed on the nodes
```

Comment:
I would be much more comfortable if this was marked as an alpha feature or similar.